judges/pipeline.py

```diff
@@ -1,13 +1,199 @@
 """Python script responsible for extracting judge data using web scraping"""
 
 from os import environ as ENV
-from time import sleep
 import logging
+from datetime import datetime
 from dotenv import load_dotenv
 import requests
 from bs4 import BeautifulSoup
+import pandas as pd
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
 
 
+# ========== FUNCTIONS: SCRAPING ==========
+def scrape_HCKB(url: str) -> pd.DataFrame:
```

Comment:
Abbreviations can be confusing, just to be clear, write the full name of the judge title we are searching for, (e.g. scrape_kings_bench_judges).